.github/workflows/ci.yml

```
@@ -140,14 +140,36 @@ jobs:
           ./ailuo install --root "$RUNNER_TEMP/pkgroot" "$tarball"
           ./ailuo list --root "$RUNNER_TEMP/pkgroot" | grep -q '^strings.tool@1.0.0'
-      - name: 打包 campus.bus（经 [build] 自动构建）并冒烟安装验证
+      - name: 打包零声明 Go 包（无 ailuo.toml，源码自动提取）并冒烟安装验证
         shell: bash
         run: |
           set -euo pipefail
-          ./ailuo pack extensions/campus.bus dist
-          tarball="$(ls dist/campus-*.tgz)"
+          mkdir -p "$RUNNER_TEMP/hello.pkg"
+          cat > "$RUNNER_TEMP/hello.pkg/go.mod" <<'EOF'
+module hello.pkg
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)